lib/msal-common/src/error/InteractionRequiredAuthError.ts

```diff
@@ -11,7 +11,8 @@ import { AuthError } from "./AuthError";
 export const InteractionRequiredServerErrorMessage = [
     "interaction_required",
     "consent_required",
-    "login_required"
+    "login_required",
+    "invalid_grant"
```

Comment:
This could cause unintended consequences, may want to remove this (from this PR, at least).

Reply:
It is a fair point though, "invalid_grant" should be translated to `UIRequired` exception. If you remove it, please open a bug / feature request to keep track of it.

Reply:
Will do @bgavrilMS